plugins/modules/rudder_server_settings.py

```diff
@@ -73,129 +72,144 @@
+        url = '/api/latest/settings/{name}'.format(name=name)
+        current_server_settings = self.get_SettingValue(name)
+        data = value
+
+        update = False
+
+        for i_settings in self.settings_to_set:
+            if (
+                current_server_settings[i_settings]
+                != self.settings_to_set[i_settings]
+            ):
+                update = True
+
+        if update:
+            self._send_request(
+                url, headers=self.headers, method='POST', data=data
+            )
 
 
 def main():
 
     module_args = dict(
-        name=dict(type="str", required=True),
+        name=dict(type='str', required=True),
         value=dict(required=True),
-        rudder_url=dict(type="str", required=False),
-        rudder_token=dict(type="str", required=False),
-        validate_certs=dict(type="bool", default=True),
+        rudder_url=dict(type='str', required=False),
+        rudder_token=dict(type='str', required=False),
+        validate_certs=dict(type='bool', default=True),
     )
 
     module = AnsibleModule(
         argument_spec={
-            "rudder_url": {"type": "str", "required": False},
-            "rudder_token": {"type": "str", "required": False},
-            "name": {"type": "str", "required": True},
-            "value": {"type": "str", "required": True},
-            "validate_certs": {"type": "bool", "default": True},
+            'rudder_url': {'type': 'str', 'required': False},
+            'rudder_token': {'type': 'str', 'required': False},
+            'name': {'type': 'str', 'required': True},
+            'value': {'type': 'str', 'required': True},
+            'validate_certs': {'type': 'bool', 'default': True},
         },
         supports_check_mode=False,
     )
 
-    rudder_url = module.params["rudder_url"]
-    rudder_token = module.params["rudder_token"]
-    name = module.params["name"]
-    value = module.params["value"]
-    validate_certs = module.params["validate_certs"]
+    rudder_url = module.params['rudder_url']
+    rudder_token = module.params['rudder_token']
+    name = module.params['name']
+    value = module.params['value']
+    validate_certs = module.params['validate_certs']
+
+    rudder_server_iface = RudderSettingsInterface(module)
+
+    VALUE = rudder_server_iface.get_SettingValue(name)
 
-    rudder_iface = RudderSettingsInterface(module)
-    VALUE = rudder_iface.get_SettingValue(name)
-    """ module.exit_json(failed=False, changed=True, message=VALUE) """
+    changed = False
 
     if str(VALUE) != value:
-        rudder_iface.set_SettingValue(name, value)
+        rudder_server_iface.set_SettingValue(name, value)
         changed = True
         module.exit_json(
-            failed=False, changed=changed, message="changed succefully"
+            failed=False,
+            changed=changed,
+            meta=module.params,
+            message='changed succefully',
```

Comment:
Could you add the new setting value in the output? uner a key `new_value` for instance